CHANGES.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+- Add scrollbar to SQL results tab bar when many resultsets are returned.
```

Comment:
I think this is duplicate with the one below (I guess the one below needs to be removed)